### PR TITLE
feat(upgrade-conversions): mark choose pro and click upgrade as conversions

### PR DESCRIPTION
**Pull request by exscruzme**

### CODE REVIEW

File: src/routes/features/+page.svelte [hunk 1]
```diff
@@ -27,7 +33,7 @@
       <div class="section-title">
         <h2>Basic</h2>
         <p>Free</p>
-        <a class="btn" href="/login">Sign Up</a>
+        <button class="btn" on:click={() => goto('/login')}><h3>Sign Up</h3></button>
```
Comment (admoya) [inline src/routes/features/+page.svelte:36]: Why is this necessary? Generally bad practice to make buttons act like links

File: src/routes/features/+page.svelte [hunk 2]
```diff
@@ -59,7 +65,7 @@
       <div class="section-title">
         <h2>Pro</h2>
         <p>$3/month</p>
-        <a class="btn" style="color: black; border-color: black;" href="/account?action=upgrade">Upgrade</a>
+        <button class="btn" style="color: black; border-color: black;" on:click={upgrade}><h3>Upgrade</h3></button>
```
Comment (admoya) [inline src/routes/features/+page.svelte:68]: I see, but I think it would be better to attach `onClick` to an `a` tag to keep it semantically correct.

Comment (exscruzme) [reply src/routes/features/+page.svelte:68]: okay ! will make that change then

Comment (exscruzme) [reply src/routes/features/+page.svelte:68]: okay so i changed the sign up link back to an anchor tag, but when changing this button to be an anchor with no `href`, a11y yells at me quite a bit lol. i think it needs to be a button so that the click handler automatically picks up any keyboard interactions.

**Review by admoya — APPROVED** [review]
